src/amber/router/router.cr

```diff
@@ -81,7 +81,12 @@ module Amber
       end
 
       def match(http_verb, resource) : Radix::Result(Amber::Route)
-        @routes.find build_node(http_verb, resource)
+        result = @routes.find build_node(http_verb, resource)
+        if result.found?
+          result
+        else
+          @routes.find build_node(http_verb, resource.sub(/\.[^$\/]+$/, ""))
```

Comment:
Can this regex be named with a constant to indicate what it is for? It is meant to chop off the extension, right? .json or .text, etc.